src/udev/udev-event.c

```diff
@@ -574,6 +576,10 @@ static int on_spawn_io(sd_event_source *s, int fd, uint32_t revents, void *userd
                                          fd == spawn->fd_stdout ? "out" : "err", *q);
         }
 
+        /* Re-enable the event source if we did not encounter EOF */
+        if (l > 0)
+                sd_event_source_set_enabled(s, SD_EVENT_ONESHOT);
```

Comment:
either explicitly cast the return value of this call to (void) or handle it by logging about it (and ignoring it otherwise). But we generally avoid just eating up error codes without casting them to (void). See https://systemd.io/CODING_STYLE#error-handling

Reply:
I have added an error log if these calls fail.